server/container/guice/mailbox/src/main/java/org/apache/james/modules/mailbox/MailboxStoreModule.java

```diff
@@ -0,0 +1,49 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.modules.mailbox;
+
+import org.apache.james.mailbox.AttachmentManager;
+import org.apache.james.mailbox.BlobManager;
+import org.apache.james.mailbox.MessageIdManager;
+import org.apache.james.mailbox.RightManager;
+import org.apache.james.mailbox.store.StoreAttachmentManager;
+import org.apache.james.mailbox.store.StoreBlobManager;
+import org.apache.james.mailbox.store.StoreMessageIdManager;
+import org.apache.james.mailbox.store.StoreRightManager;
+import org.apache.james.mailbox.store.StoreSubscriptionManager;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Scopes;
+
+public class MailboxStoreModule extends AbstractModule {
```

Comment:
MailboxStore doesn't mean anything to me. Could we find a better description? Like DefaultMailboxSomething? (It's hard to replace "Something" because responsibility is not clearly defined AFAIU)